algorithm_catalog/max_ndvi_composite_description.md

```diff
@@ -0,0 +1,71 @@
+# Description
+
+This algorithm generates a Sentinel-2 based composite for a selected area and temporal extent. By default, the resolution of the output is 10 meters.
+
+The used compositing method is the "max-NDVI" method, which selects the pixel with the highest NDVI value for each pixel location and within the time window.
+The method falls under the 'rank composite' category, and ensures that selected spectral band values for any individual pixel all come from the same observation.
+
+The method generates good results for timeseries analytics, but spatially neighbouring pixels may be selected from different observations,
+which leads to visual discontinuities in the result.
+
+# Performance characteristics
+
+The method is computationally efficient, as it only requires the B04, B08 and SCL bands to determine the rank score. Loading 
+of other bands can be minimized to read only selected observations.
+
+
+# Examples
+
+The image below shows a typical result over an agricultural area.
+
+![max_ndvi_example.png](max_ndvi_composite_files/max_ndvi_example.png)
+
+A complete example including STAC metadata is shown here:
+
+https://radiantearth.github.io/stac-browser/#/external/s3.waw3-1.cloudferro.com/swift/v1/APEx-examples/max_ndvi_denmark/collection.json
+
+The processing platform reported these usage statistics for the example:
+
+```
+Credits: 63 
+CPU usage: 47.743,722 cpu-seconds
+Wall time: 1.948 seconds
+Input Pixel 10.997,635 mega-pixel
+Max Executor Memory: 3,239 gb
+Memory usage: 154.767.121,977 mb-seconds
+Network Received: 1.677.537.930.040 b
+```
+
+The relative cost is 1 platform credits per km² for a 3 month input window.
```

Comment:
is this an openEO platform credit (also CDSE?)? might be worth noting, as other credits might not be comparable...

Reply:
I added cdse explicitly. The link with the platform is also made in the metadata json.
Ideally, when the web portal renders our algorithm catalog, it should clearly show which algorithms are (officially) associated with which platform.
Also good to note that this algorithm will in fact work just fine on openEO platform, only thing is that SENTINEL2_L2A might come in via sentinelhub from aws, which fill inflate credit cost a lot.